.github/workflows/build.yml

```diff
@@ -3,7 +3,15 @@ name: Build
 on:
   push:
     branches: [master]
-  pull_request: ~
+    paths-ignore:
+      - 'CHANGELOG.md'
```

Comment:
Perhaps we could just use '- '**.md' to cover all markdown files?

Reply:
better idea ! 

done.